src/Symfony/Bundle/FrameworkBundle/Command/CachePooltemDeleteCommand.php

```diff
@@ -0,0 +1,81 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Bundle\FrameworkBundle\Command;
+
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Style\SymfonyStyle;
+use Symfony\Component\HttpKernel\CacheClearer\Psr6CacheClearer;
+
+/**
+ * Delete an item from a cachhe pool.
+ *
+ * @author Pierre du Plessis <pdples@gmail.com>
+ */
+final class CachePooltemDeleteCommand extends Command
```

Comment:
missing `I` before `tem`, nice typo :)

Reply:
but this doesn't match the name of the command
`cache:pool:delete` LGTM, so the previous name was OK IMHO